shared-helpers/src/utilities/pdfs.ts

```diff
@@ -31,7 +31,7 @@ export const cloudinaryPdfFromId = (publicId: string, cloudName: string) => {
  */
 export const getPdfUrlFromAsset = (
   asset: AssetCreate,
-  cloudinaryCloudName = process.env.CLOUDINARY_CLOUD_NAME || ""
+  cloudinaryCloudName = process.env.CLOUDINARY_CLOUD_NAME || "exygy"
```

Comment:
Are we sure we want to put "exygy" in here rather than have it pulling from the .env file where it currently exists?

Reply:
This is maybe a bigger discussion, but the short version is that the only time the `cloudinaryCloudName` var will be used is when we're showing listings from the other Bloom portal, which only ever uses the "exygy" cloud name and the configuration of which resides entirely on their end.  We can override it with our own env vars if we need to (with some limitations imposed by Next), but it's unlikely to change between now and the time those portals are deprecated.

Reply:
Got it.